css/style.css

```diff
@@ -16,4 +35,83 @@ body {
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
 
+* {
+    box-sizing: border-box;
+    font-size: 100%;
+    margin: 0;
+    border: 0;
+    padding: 0;
+    font-family: inherit;
+}
+
+body {
+    line-height: 1;
+}
+
+
+/* Nav section styling */
+
+.navigation {
+    width: 100%;
+    background-color: pink;
+    position: fixed;
+    z-index: 9999;
+}
+
+.navigation .links {
+    float: right;
+}
+
+.navigation .links li {
+    display: inline-block;
+    color: #838994;
+}
+
+.bold {
+    font-weight: bold;
+}
+
+.bold hover {
+    background-color: bisque;
+}
+
+.button {
+    width: 500px;
+    height: 200px;
+    padding: 40px 60px;
+}
+
+
+/* Introduction styling */
+
+.introduction {
+    background-image: url(../img/first-background.jpg);
+    height: 45.938rem;
+}
+
+
+/* Boxes styling */
+
+.features {
+    display: flex;
+    justify-content: center;
+}
```

Comment:
This is good! Can you add a media query for these list-items inside to appear in a column on mobile?

A media query looks like this: 
``` 
@media (max-width: 700px) {
    .features {
      // rule to make it into a column here (use flexbox!)
    }
  }
```

Reply:
In this particular example, this media query would set a CSS rule that would apply _only_ when the screen is smaller than 700px wide. 
In other words, the CSS you've already written would work on desktop (leave it as it is), and the CSS inside this media query would work on screens smaller than 700px.